packages/rum/src/domain/record/serialize.ts

```diff
@@ -30,18 +24,29 @@ import {
 } from './serializationUtils'
 import { forEach } from './utils'
 
+// Those values are the only one that can be used when inheriting privacy levels from parent to
+// children during serialization, since HIDDEN and IGNORE shouldn't serialize their children. This
+// ensures that no children are serialized when they shouldn't.
+type ParentNodePrivacyLevel =
```

Comment:
Looks like this is the same as the `InitialPrivacyLevel` type

Reply:
Yes, but it is somewhat coincidental. The `InitialPrivacyLevel` is the privacy levels that we want to expose publicly, and `ParentNodePrivacyLevel` are the privacy levels that can be inherited during serialization, so they may not always be the same in the future. To avoid confusion, I'd keep those two separated.

Reply:
Makes sense.